src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java

```diff
@@ -129,8 +129,6 @@ public class PrlAppsConstants {
                                                      "caseworker-privatelaw-judge",
                                                      "caseworker-privatelaw-la");
 
-    public static final List<String> ROLES_JUDGE = List.of("caseworker-privatelaw-judge",
-                                                     "caseworker-privatelaw-la");
```

Comment:
what happened to this?

Reply:
It is already moved to an ENUM as discussed earlier